tests/BenchmarkDotNet.Tests/Exporters/JsonExporterTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Exporters.Json;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Tests.Mocks;
+using Xunit;
+using JsonSerializer = SimpleJson.SimpleJson;
+
+namespace BenchmarkDotNet.Tests.Exporters
+{
+    public class JsonExporterTests
+    {
+        [Fact]
+        public void JsonExporter_ExportToLog_ReturnsValidJson()
+        {
+            //Arrange
+            var logger = new AccumulationLogger();
+            var summary = MockFactory.CreateSummary(DefaultConfig.Instance);
+
+            //Act
+            JsonExporter.Default.ExportToLog(summary, logger);
+
+            //Assert
+            Assert.True(JsonSerializer.TryDeserializeObject(logger.GetLog(), out object obj));
+        }
```

Comment:
we already have tests that check the Json serializer output: https://github.com/dotnet/BenchmarkDotNet/blob/9744089383f881506c8069f9802260e63c448ea1/tests/BenchmarkDotNet.Tests/Exporters/CommonExporterApprovalTests.cs#L85-L88

Could you please remove this test, make `ReplaceUnsupportedNumericValues` internal and just test what it returns for valid and invalid inputs?

Reply:
Some tests will be failed due hardcoded `NaN` values in text files from ApprovedFiles folder.
Please, review new changes.

Reply:
@marcnet80 the new changes look good to me! but we need to get the CI green.

The easiest way of updating the approved files after on purpose exporter changes is:

* go to `tests\BenchmarkDotNet.Tests`
* run the tests `dotnet test -c Release -f netcoreapp2.1`
* go to `Exporters\ApprovedFiles` and check which `.approved` file is different than `.received`. Ensure that the changes are what we want and remove the old `.approved` file and rename the `.received` file `.approved` 
* re-run the tests
* if everything is green, push the changes

![obraz](https://user-images.githubusercontent.com/6011991/98093426-3c3dca00-1e88-11eb-90bc-09cb5fef1ed3.png)

Reply:
fixed.